server/app/controllers/dev/seeding/DevDatabaseSeedController.java

```diff
@@ -93,11 +105,23 @@ public Result clear() {
     if (!isDevOrStaging) {
       return notFound();
     }
+    if (settingsManifest.getVersionCacheEnabled()) {
+      clearCache();
+    }
     resetTables();
     return redirect(routes.DevDatabaseSeedController.index().url())
         .flashing("success", "The database has been cleared");
   }
 
+  /** Remove all content from the cache. */
+  public void clearCache() {
+    if (!isDevOrStaging) {
+      return;
```

Comment:
**nitpick:** this is already guarded in the action before this method gets called

Reply:
Yeah, that's true - removed!